### ISSUE TITLE
Baseline Discovery: Subscribe CTAs

### ISSUE THREAD

**Issue by anthonycuellar** [NCIOCPL/ncids#129]
Gain context around behavior and use of "subscribe" CTAs:
- **Subscribe link on blog series** 
  - https://www.cancer.gov/news-events/cancer-currents-blog
  - https://www.cancer.gov/grants-training/nci-bottom-line-blog
  - https://www.cancer.gov/about-nci/organization/cgh/blog
  - https://www.cancer.gov/about-nci/organization/crchd/blog
  - https://www.cancer.gov/about-nci/organization/ccg/blog
- **Email signup callouts on pages across Cgov**
  - Article - callout at top of page with OTP: https://www.cancer.gov/about-cancer/understanding/what-is-cancer
  - Article, callout at top of page: https://www.cancer.gov/research/key-initiatives/moonshot-cancer-initiative/funding
  - Navigational Article, for conferences: https://www.cancer.gov/news-events/events/conferences
  - Social media events page (not necessarily in-depth content): https://www.cancer.gov/news-events/events/social-media
  - PDQ index pages: https://www.cancer.gov/publications/pdq (article) and https://www.cancer.gov/publications/pdq/information-summaries (mini landing). Also look at subpages here.

**Some Questions:**
- Are people clicking the link/button?
  - Need to determine if there any button CTAs (see tasks below) and if so, can we determine if one is more successful than the other? I.e., do people click the button CTA more than the link version, or vice versa?
- Are those who click actually subscribing to a newsletter (are we able to determine if they complete a GovDelivery signup?)
- Is there a disconnect between the terminology (subscribe) and the action (a general signup for newsletters) 

Findings should be captured in the Notion for discussion.

**Generated from Discovery Kickoff: Blog Post #73**

## Tasks

- [x]  Identify pages with email signup CTA, denoting whether the CTA is a link or a button
- [ ]  Synthesis of any existing analytics Data around use and function - click rates of link/button
- [x]  High-level heuristic review of feature
- [ ]  Technical or accessibility flags
- [ ]  Best practices around use of features

### Resources

- Notion - Blog Posts - https://www.notion.so/Blog-Post-280acddd57694d90a299701fa3cd4326
- Related Ticket: Past research on use of button vs. text link for blog subscriptions #138 

### Notes

*Some complementary notes if necessary:*

**Comment by nicolecrom** [comment]
I've added analytics on blog subscription to notion
In order to pull analytics for other CTA, I will need more info on what pages have CTAs.

**Comment by mworrest** [comment]
Analysis performed on Min's report: https://lion.box.com/s/qy7sfpok8mignk3mvlzf9mz25767a1uo
I found that 37 of the URLs in the report are legitimate. The others are either links within the body or broken URLs. I think this report is accurate, albeit a low number of pages. I added my notes to the document on box. The blog landing pages seem to be excluded, but we have an understanding that the subscribe link comes standard on every blog and blog series.
